source/routing/tests/reverse_raptor_test.cpp

```diff
@@ -35,6 +35,12 @@ www.navitia.io
 
 #include "routing/raptor.h"
 #include "ed/build_helper.h"
+#include "tests/utils_test.h"
+
+struct logger_initialized {
+    logger_initialized() { init_logger(); }
+};
+BOOST_GLOBAL_FIXTURE( logger_initialized )
```

Comment:
please add a `;` at the end (required in later versions of boost, even if it warns on double `;` in 1.55)